code/modules/client/preferences.dm

```diff
@@ -19,6 +19,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	//game-preferences
 	var/lastchangelog = ""				//Saved changlog filesize to detect if there was a change
 	var/ooccolor = null
+	var/enable_tips = TRUE

```

Comment:
Bump This number: https://github.com/tgstation/tgstation/blob/master/code/modules/client/preferences_savefile.dm#L5
Preference Loading: https://github.com/tgstation/tgstation/blob/master/code/modules/client/preferences_savefile.dm#L117
Preference Saving: https://github.com/tgstation/tgstation/blob/master/code/modules/client/preferences_savefile.dm#L186